src/components/timeline/TrackMemoryGraph.js

```diff
@@ -33,7 +33,7 @@ import type {
 import type { SizeProps } from 'firefox-profiler/components/shared/WithSize';
 import type { ConnectedProps } from 'firefox-profiler/utils/connect';
 
-import './TrackMemory.css';
+import 'firefox-profiler/components/timeline/TrackMemory.css';
```

Comment:
Please revert this line, it starts with `./` and it's not related to `src/actions`.